src/lib/serviceRoutinesV2/getEntityAllTypes.cpp

```diff
@@ -84,7 +84,6 @@ std::string getEntityAllTypes
                                  totalTypesP,
                                  noAttrDetail));
   }
-  TIMED_RENDER(answer = response.toJson(ciP->uriParamOptions[OPT_VALUES]));
```

Comment:
Why are you removing this line? From my point of view, it should be kept.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/397ae9e496a9ce542907b10016aa464b69ba71a3